pkg/k8s/pipelineRun.go

```diff
@@ -280,14 +284,13 @@ func (r *pipelineRun) InitState() error {
 }
 
 // UpdateState implements part of interface `PipelineRun`.
-func (r *pipelineRun) UpdateState(state api.State, timestamp metav1.Time) error {
+func (r *pipelineRun) UpdateState(ctx context.Context, state api.State, timestamp metav1.Time) error {
 	if r.apiObj.Status.State == api.StateUndefined {
-		if err := r.InitState(); err != nil {
+		if err := r.InitState(ctx); err != nil {
 			return err
 		}
 	}
 	r.ensureCopy()
-	klog.V(3).Infof("Update State to %s [%s]", state, r.String())
```

Comment:
Why is this dropped? May be change to verbosity 4?

Reply:
I have already taken care of it in caller-side code - [pkg/runctl/controller.go](https://github.com/SAP/stewardci-core/pull/387/files/eca51205dce2700fd9e3aaada7d6ecbb90439cdb#diff-910fd61456d9e06cc4eea86430c974b6db8ca8fe956dfdfb8a0a43b0c680272eR254).